app/javascript/components/organization/EditProfileForm.jsx

```diff
@@ -0,0 +1,263 @@
+import React from "react";
+import axios from "axios";
+import Dropdown from '../../utils/Dropdown';
+
+class EditProfileForm extends React.components {
+  constructor(props) {
+    super(props);
+    var name = props.name;
+    var website = props.organization.website;
+    var location = props.organization.lcoation;
+    var passwrod = "*****";
```

Comment:
misspelling of password. if this was taken from the `EditProjectForm`, then this is my fuk up haha.